digital-ocean/digital-ocean.lua

```diff
@@ -0,0 +1,172 @@
+--[[
+
+# [Digital Ocean API](https://developers.digitalocean.com/documentation/v2/) connector.
+
+### Authorization
+
+Authorization use [API Key](https://cloud.digitalocean.com/settings/applications).
+
+## Input parameters
+
+* api_key - API Key.
```

Comment:
Rename this to `access_token` this is a part of OAuth authorization, even you may pass it the username in digital ocean implementation, but its confusing, please send this over standard "Authorization" header.